src/Powercord/plugins/pc-spotify/commands/like.js

```diff
@@ -0,0 +1,37 @@
+const playerStore = require('../playerStore/store');
+const SpotifyAPI = require('../SpotifyAPI');
+
+module.exports = {
+  command: 'like',
+  description: 'Like a current playing song',
+  async executor () {
+    const currentTrack = playerStore.getCurrentTrack();
+    if (!currentTrack) {
+      return {
+        send: false,
+        result: 'You are not currently listening to anything.'
+      };
+    }
+    const hasCoolFeatures = powercord.account && powercord.account.spotify;
+    if (!hasCoolFeatures) {
+      return {
+        send: false,
+        result: 'You need a Powercord account and connected Spotify account.'
+      };
+    }
```

Comment:
It'd be better to check for the Powercord account first
```suggestion
    if (!powercord.account && powercord.account.spotify) {
      return {
        send: false,
        result: 'You need a Powercord account and connected Spotify account.'
      };
    }
    const currentTrack = playerStore.getCurrentTrack();
    if (!currentTrack) {
      return {
        send: false,
        result: 'You are not currently listening to anything.'
      };
    }
```